### DISCUSSION TITLE
layouts are braking at different browser

### DISCUSSION THREAD

**Discussion by Alam487** [quasarframework/quasar#10207] [General - Components / Directives / etc]
[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/MuFIK.png

I am using quasar frame work for making my site responsive and everything is working fine. I have tested in mobile responsive it is working fine And I have also tested in different browser it is working fine but coming to different platform layout are breaking like if i am testing at windows
laptop everything is fine but while i am testing at mac laptop layout are braking i am not able to understand where is the problem it is related to a browser or related to OS.

*quasar links*

```
<head>
    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900|Material+Icons" rel="stylesheet" type="text/css">
    <link href="https://cdn.jsdelivr.net/npm/quasar@2.0.3/dist/quasar.prod.css" rel="stylesheet" type="text/css">
  </head>
```   

```
<script src="https://cdn.jsdelivr.net/npm/vue@3/dist/vue.global.prod.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/quasar@2.0.3/dist/quasar.umd.prod.js"></script>
```
*config object*

```
app.use(Quasar, {
  config: {
    brand: { // this will NOT work on IE 11
      primary: '#e46262',
      // ... or all other brand colors
    },
    notify: {...}, // default set of options for Notify Quasar plugin
    loading: {...}, // default set of options for Loading Quasar plugin
    loadingBar: { ... }, // settings for LoadingBar Quasar plugin
    // ..and many more
  }
})
```

**Comment by MilosPaunovic** [comment]
Have you checked screen resoultions, browser tab zoom in/out...

**Reply by Alam487** [reply]
Yes I have checked. the issues I have faced like I have given a css color to a <p> tag. SO in windows laptop or chrome browser it is fine but coming to mac laptop the color is changing

**Reply by MilosPaunovic** [reply]
You've tried in the same browsers in Windows and MacOS?
Which versions of OS and browsers?

**Reply by Alam487** [reply]
yes I have tried in same browser in windows and mac so that is Firefox.

**Reply by MilosPaunovic** [reply]
Please make a codepen that can reproduce the problem and provide exact version of browsers in which the problem is appearing so someone could debug it.
